release/release.md

```diff
@@ -0,0 +1,122 @@
+# How to do a Kata Containers Release
+
+* [Introduction](#Introduction)
+* [Requirements](#Requirements)
+* [Release process](#release-process)
+
+# Introduction
+
+If you are reading this document, you might want to create a Kata Containers
+Release.
+
+The Kata Containers Release Process is defined in the following
+[document][release-process-definition]. To simplify this process, we have
+created a [Release Checklist][release-checklist]. Finally, to simplify the
+Release Checklist process we have automated most of the process, this document
+guides on how to use release scripts instead of do all the checklist manually.
+
+
+## Requirements
+
+- It is recommended to have at least 2 GB of free disk space to perform these tasks.
+
+- [Go](https://golang.org/doc/install#)
+
+- [hub](https://github.com/github/hub)
+  If you use GitHub 2FA, configure the hub manually to create a file ~/.config/hub
```

Comment:
> ... configure hub by creating file `~/.config/hub`:

Since this contains a secret, we should probably recommend users set strict perms on it:

```sh
$ chmod 400 ~/.config/hub
```

Reply:
Removed, I've seen that now supports ask for 2 step code, not needed to do this manually again. I just tested and is working.